Sources/SentryCrash/Recording/Monitors/SentryCrashMonitorType.h

```diff
@@ -92,8 +94,6 @@ typedef enum {
 #define SentryCrashMonitorTypeRequired                                                             \
     (SentryCrashMonitorTypeSystem | SentryCrashMonitorTypeApplicationState)
 
-#define SentryCrashMonitorTypeNone 0
```

Comment:
This wouldn't work in the new swift implementation of an objc impl that was using this, so i changed it to be a member of the enum instead.